### PR TITLE
streamsx.topology.view changes to be made in conjunction with changes in streamsx.utility

### PR DESCRIPTION
**Pull request by wmarshall484**
* [PR 26](https://github.com/IBMStreams/streamsx.utility/pull/26) in streamsx.utility adds view threading support to streamsx.rest_primitives.View. Therefore, streamsx.topology.view only needs to act as a wrapper.  

* This PR should be merged only after [PR 26](https://github.com/IBMStreams/streamsx.utility/pull/26) is merged.

### CODE REVIEW

File: com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/errors.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+class ViewNotFoundError(Exception):
```
Comment (ddebrunner) [inline com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/errors.py]: Why do we need two View not found errors?

File: com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/topology.py [hunk 2]
```diff
@@ -508,25 +509,17 @@ class Routing(Enum):
     HASH_PARTITIONED=3    
 
 
-class View(threading.Thread):
+class View(object):
```
Comment (ddebrunner) [inline com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/topology.py]: Why not extend the `streamsx.rest` View object?

File: com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/topology.py [hunk 3]
```diff
@@ -544,88 +537,23 @@ def initialize_rest(self):
...
-    def __call__(self):
-        while not self._stopped():
-            time.sleep(1)
-            _items = self._get_view_items()
-            if _items is not None:
-                for itm in _items:
-                    self.items.put(itm)
+        return self.view_object.start_data_fetch()
 
     def set_streams_context_config(self, conf):
```
Comment (ddebrunner) [inline com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/topology.py]: Doesn't seem like these are external methods, should they be `_set...`

**Comment by wmarshall484** [comment]
Updates on this PR and [PR 26 in streamsx.topology](https://github.com/IBMStreams/streamsx.utility/pull/26)

Changes.
* Supplying a VCAP config to a StreamsContext constructor is now supported
* Python REST API test harness has been updated to allow tests to run on Bluemix
* Threading logic has been separated from rest_primitives.View into rest_primitives.ViewThread
* StreamsContext.retrieve_views() and StreamsContext.retrieve_views() methods have been added.

TODO/comments:
* Document methods and class
* Use vcap processing from rest api
* Remove unnecessary methods in topology.View
* Should topology.View inherit from rest_primitives.View?

**Comment by ddebrunner** [comment]
> Should topology.View inherit from rest_primitives.View?

Maybe also why is a new class required at all? What is in the topology View that is not in the rest view?

**Pull request merged**